test/character_test.rb

```diff
@@ -0,0 +1,25 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/character'
+
+class CharacterTest < Minitest::Test
+  def setup
+    @kitt = Character.new({name: "KITT", actor: "William Daniels", salary: 1_000_000})
+  end
+
+  def test_character_instance
+    assert_instance_of Character, @kitt
+  end
+
+  def test_character_name
+    assert_equal 'KITT', @kitt.name
+  end
+
+  def test_character_actor
+    assert_equal "William Daniels", @kitt.actor
+  end
+
+  def test_character_salary
+    assert_equal 1000000, @kitt.salary
+  end
```

Comment:
good work in here; it's okay to test all initialized attributes in a single test